deploy/config/nginx.conf

```diff
@@ -8,6 +8,9 @@ server {
     listen 1443;
     client_max_body_size 4G;
 
+    access_log  /var/log/nginx/access.log;
+    error_log /var/log/nginx/debug.log debug;
```

Comment:
@deepakdinesh1123 can we change the file name to `error.log` and not specify any log level? by default it will include error, crit, alert, emerg logs